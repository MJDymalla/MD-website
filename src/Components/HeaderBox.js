import '../CSS/header_box.css';

import React, {Component} from 'react';

import M from 'materialize-css';
import header_image from '../Images/header_image.jpg';

class HeaderBox extends Component {
    componentDidMount() {
        M.Parallax.init(this.Parallax1);
    }
    render() {
        return (
            <div className="header-img-height">
                <div className="parallax-container header-img-container">
                    <div
                        ref={Parallax => {
                            this.Parallax1 = Parallax;
                        }}
                        className="parallax"
                    >
                        <div>
                            <img src={header_image} alt=""/>
                        </div>
                    </div>
                </div>
            </div>
        );
    }
}
export default HeaderBox;