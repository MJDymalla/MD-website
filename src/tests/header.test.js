import React from 'react';
import ReactDOM from 'react-dom';
import ShallowRenderer from 'react-shallow-renderer'; 
import HeaderBox from '../Components/HeaderBox';

test('should render header correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<HeaderBox />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
})




