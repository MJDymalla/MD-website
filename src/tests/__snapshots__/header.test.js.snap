// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div
  className="header-img-height"
>
  <div
    className="parallax-container header-img-container"
  >
    <div
      className="parallax"
    >
      <div>
        <img
          alt=""
          src="header_image.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
